puppet/modules/profiles/manifests/redmine.pp

```diff
@@ -4,12 +4,47 @@
   # Redmine is already included via ENC
   # TODO include redmine
   $backup_path = '/var/lib/redmine'
+  $backup_db_path = "${backup_path}/db"
```

Comment:
`restic` can't access `/var/lib/redmine` right now, so we would need to either place the dump somewhere else, or allow restic access, feels odd, both.

Reply:
We give it CAP_DAC: doesn't that mean it can override it?

Reply:
indeed :see_no_evil: 

I was testing this outside systemd, and thus had no CAP mangling